EIPS/eip-5606.md

```diff
@@ -0,0 +1,114 @@
+---
+eip: 5606
+title: Proxy NFTs
+description: A universal representation of a digital asset across various platforms through a wrapper NFT
```

Comment:
Hm... So most NFTs are already representations of digital assets, so perhaps mention something more specific to the use cases you mentioned on the EIP Office Hours call.